### ISSUE TITLE
Duplicate species in a model?

### ISSUE THREAD

**Issue by mjohnson541** [ReactionMechanismGenerator/RMG-Py#1221]
Conducting simulation of reaction system 1...
initializing surface ...
surface initialization complete
At time 7.4247e-09 s, PDepNetwork #5 at 0.0117306639562 exceeded the minimum rate for simulation interruption of 0.01
Reached max number of objects...preparing to terminate
At time 7.4247e-09 s, PDepNetwork #5 at 0.0117306639562 exceeded the minimum rate for exploring of 0.01
terminating simulation due to interrupt...
Making seed mechanism...
Traceback (most recent call last):
  File "/Users/mattjohnson/RMGCODE/RMG-Py/rmg.py", line 173, in <module>
    main()
  File "/Users/mattjohnson/RMGCODE/RMG-Py/rmg.py", line 167, in main
    rmg.execute(**kwargs)
  File "/Users/mattjohnson/RMGCODE/RMG-Py/rmgpy/rmg/main.py", line 637, in execute
    self.makeSeedMech()
  File "/Users/mattjohnson/RMGCODE/RMG-Py/rmgpy/rmg/main.py", line 841, in makeSeedMech
    shortDesc = species.thermo.comment
  File "/Users/mattjohnson/RMGCODE/RMG-Py/rmgpy/data/thermo.py", line 604, in loadEntry
    raise DatabaseError('Adjacency list and multiplicity of {0} matches that of existing molecule {1} in thermo library {2}.  Please correct your library.'.format(label, entry.label, self.name))
rmgpy.exceptions.DatabaseError: Adjacency list and multiplicity of O2-2 matches that of existing molecule O2 in thermo library Syngas.  Please correct your library.  

I got this error running a syngas example on master.  I haven't yet been able to duplicate this in one of the standard examples yet, but I haven't tried all of them.  I've confirmed in the dictionary file that the O2's (O2(3) and O2(15)) are identical.  I also got a curious adjacency list for OO:  

OO
1 O u0 p1 c+1 {2,S} {3,S} {4,S}
2 O u0 p3 c-1 {1,S}
3 H u0 p0 c0 {1,S}
4 H u0 p0 c0 {1,S}

**Comment by mjohnson541** [comment]
Also 
HO2(6)
multiplicity 2
1 H u0 p0 c0 {2,S}
2 O u1 p1 c+1 {1,S} {3,S}
3 O u0 p3 c-1 {2,S}

**Comment by mjohnson541** [comment]
I assume these adjacency lists are a result of the lone pair resonance stuff, but I'm not sure why it's not realizing that the second O2 is the same as the original O2.

**Comment by alongd** [comment]
The duplicate O2 species in this issue are:
```
multiplicity 3
1 O u1 p2 c0 {2,S}
2 O u1 p2 c0 {1,S}
```
```
multiplicity 3
1 O u0 p3 c-1 {2,S}
2 O u2 p1 c+1 {1,S}
```

As discussed offline with @mliu49 and @mjohnson541, the  extended LonePairRadical resonance commit (b88c9f7) that causes this isn't symmetric in exploring the paths and cases a hysteresis.

**Comment by mliu49** [comment]
Fixed by #1223.
